desk/app/chat.hoon

```diff
@@ -920,6 +962,24 @@
       (cu-post-notice for.delta '' ' was uninvited from the chat') 
     ==
   ::
+  ++  cu-remark-diff
+    |=  diff=remark-diff:c
+    ^+  cu-core
+    =.  remark.club
+      ?-  -.diff
+        %watch    remark.club(watching &)
+        %unwatch  remark.club(watching |)
+        %read-at  !! ::  cu-core(last-read.remark.chat p.diff)
+      ::
+          %read
+      =/  =time
+        (fall (bind (ram:on:writs:c wit.pact.club) head) now.bowl)
+      remark.club(last-read `@da`(add time 1))  ::  greater than last
```

Comment:
hmm we should probably replace this (and similar invocations) with a 10ms buffer instead: `(add time (div ~s1 100))`, since unix timestamp conversion will cause some problems:

```hoon
> [`@`now (add now 1)]
170.141.184.506.011.029.893.607.972.542.502.404.096
170.141.184.506.011.029.893.607.972.542.502.404.097
```

```javascript
daToUnix(bigInt(udToDec('170.141.184.506.011.029.893.607.972.542.502.404.096')))
===
daToUnix(bigInt(udToDec('170.141.184.506.011.029.893.607.972.542.502.404.097')))
true
```
with 10ms buffer instead:

```hoon
> [`@`now (add now (div ~s1 100))]
170.141.184.506.011.035.979.142.005.023.158.829.056
170.141.184.506.011.035.979.326.472.463.895.924.572
```

```javascript
daToUnix(bigInt(udToDec('170.141.184.506.011.035.979.142.005.023.158.829.056')))
===
daToUnix(bigInt(udToDec('170.141.184.506.011.035.979.326.472.463.895.924.572')))
false
```
```
[daToUnix(bigInt(udToDec('170.141.184.506.011.035.979.142.005.023.158.829.056'))),
 daToUnix(bigInt(udToDec('170.141.184.506.011.035.979.326.472.463.895.924.572')))]
(2) [1672862587475, 1672862587485]
```